nucliadb_protos/python/nucliadb_protos/nodesidecar_pb2_grpc.pyi

```diff
@@ -12,6 +12,7 @@ from nucliadb_protos.noderesources_pb2 import (
     IndexMetadata as IndexMetadata,
     IndexParagraph as IndexParagraph,
     IndexParagraphs as IndexParagraphs,
+    NodeMetadata as NodeMetadata,
```

Comment:
does it belong to this PR?

Reply:
No, it's missing from management of entities. I probably forgot to rebuild protos after one of the last rebases